app/controllers/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+ class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def edit
+    #puts params
+
+    @user = User.find_by_id params[:id]
```

Comment:
Since .find is already finding its id, maybe we could refactor it to User.find params[:id] instead? But it works either way